core/application/impl/validating_node_application.hpp

```diff
@@ -76,14 +74,14 @@ namespace kagome::application {
         uint8_t verbosity);
 
     void run() override;
-    void shutdown() override;
 
    private:
     // need to keep all of these instances, since injector itself is destroyed
     InjectorType injector_;
 
-    sptr<boost::asio::io_context> io_context_;
-	  std::unique_ptr<boost::asio::signal_set> signals_;
+	  std::shared_ptr<AppStateManager> app_state_manager_;
```

Comment:
formatting

Reply:
Methods defined in class declaration is inline defaultly.

Reply:
Fixed